os/board/rtl8721csm/src/component/os/tizenrt/rtk_blemgr.c

```diff
@@ -0,0 +1,419 @@
+trble_result_e trble_netmgr_operation_write_no_response(struct bledev *dev, trble_operation_handle *handle, trble_data *in_data);
+
+/*** Peripheral(Server) ***/
+trble_result_e trble_netmgr_get_profile_count(struct bledev *dev, uint16_t *count);
+trble_result_e trble_netmgr_charact_notify(struct bledev *dev, trble_attr_handle attr_handle, trble_conn_handle con_handle, trble_data *data);
+trble_result_e trble_netmgr_attr_set_data(struct bledev *dev, trble_attr_handle attr_handle, trble_data *data);
+trble_result_e trble_netmgr_attr_get_data(struct bledev *dev, trble_attr_handle attr_handle, trble_data *data);
+trble_result_e trble_netmgr_attr_reject(struct bledev *dev, trble_attr_handle attr_handle, uint8_t app_errorcode);
+trble_result_e trble_netmgr_get_mac_addr_by_conn_handle(struct bledev *dev, trble_conn_handle con_handle, uint8_t bd_addr[TRBLE_BD_ADDR_MAX_LEN]);
+trble_result_e trble_netmgr_get_conn_handle_by_addr(struct bledev *dev, uint8_t bd_addr[TRBLE_BD_ADDR_MAX_LEN], trble_conn_handle *con_handle);
+trble_result_e trble_netmgr_set_adv_data(struct bledev *dev, trble_data *data);
+trble_result_e trble_netmgr_set_adv_resp(struct bledev *dev, trble_data *data);
+trble_result_e trble_netmgr_conn_is_active(struct bledev *dev, trble_conn_handle con_handle, bool *is_active);
+trble_result_e trble_netmgr_conn_is_any_active(struct bledev *dev, bool *is_active);
+trble_result_e trble_netmgr_get_bonded_device(struct bledev *dev, trble_bonded_device_list_s *device_list, uint16_t *device_count);
+trble_result_e trble_netmgr_start_adv(struct bledev *dev);
+trble_result_e trble_netmgr_start_adv_directed(struct bledev *dev, uint8_t bd_addr[TRBLE_BD_ADDR_MAX_LEN]);
+trble_result_e trble_netmgr_stop_adv(struct bledev *dev);
+trble_result_e trble_netmgr_set_adv_interval(struct bledev *dev, uint16_t interval);
+
+struct trble_ops g_trble_drv_ops = {
+	// Common
+	trble_netmgr_init,
+	trble_netmgr_deinit,
+	trble_netmgr_get_mac_addr,
+	trble_netmgr_disconnect,
+	trble_netmgr_delete_bond_all,
+
+	// Client
+	trble_netmgr_start_scan,
+	trble_netmgr_stop_scan,
+	trble_netmgr_connect,
+	trble_netmgr_disconnect_all,
+	trble_netmgr_connected_device_list,
+	trble_netmgr_connected_info,
+	trble_netmgr_operation_enable_notification,
+	trble_netmgr_operation_read,
+	trble_netmgr_operation_write,
+	trble_netmgr_operation_write_no_response,
+
+	// Server
+	trble_netmgr_get_profile_count,
+	trble_netmgr_charact_notify,
+	trble_netmgr_attr_set_data,
+	trble_netmgr_attr_get_data,
+	trble_netmgr_attr_reject,
+	trble_netmgr_get_mac_addr_by_conn_handle,
+	trble_netmgr_get_conn_handle_by_addr,
+	trble_netmgr_set_adv_data,
+	trble_netmgr_set_adv_resp,
+	trble_netmgr_conn_is_active,
+	trble_netmgr_conn_is_any_active,
+	trble_netmgr_get_bonded_device,
+	trble_netmgr_start_adv,
+	trble_netmgr_start_adv_directed,
+	trble_netmgr_stop_adv,
+	trble_netmgr_set_adv_interval,
+};
+
+#include <stdio.h>

```

Comment:
How about moving this include into the line 25?

Reply:
@JeonginKim 
It was just for test.  I will remove it